qiskit/providers/ibmq/experiment/experimentservice.py

```diff
@@ -490,28 +494,34 @@ def upload_plot(
                 plot_name = plot
             else:
                 plot_name = "plot_{}.svg".format(datetime.now().isoformat())
-        return self._api_client.experiment_plot_upload(experiment, plot, plot_name)
+        return self._api_client.experiment_plot_upload(experiment, plot, plot_name,
+                                                       sync_upload=sync_upload)
 
     def update_plot(
             self,
             experiment: Union[Experiment, str],
             plot: Union[str, bytes],
-            plot_name: str
+            plot_name: str,
+            sync_upload: bool = True
     ) -> Dict:
         """Update an experiment plot.
 
         Args:
             experiment: The ``Experiment`` object or the experiment UUID.
             plot: Name of the plot file or plot data to upload.
             plot_name: Name of the plot to update.
+            sync_upload: By default the server will upload the plot file
+                to backend storage asynchronously. Set this to False to use
+                that behavior and not block the upload.
```

Comment:
```suggestion
            sync_upload: If ``True``, the plot will be uploaded synchronously. 
                Otherwise the upload will be asynchronous. 
```